src/test/java/seedu/address/storage/JsonAdaptedPersonTest.java

```diff
@@ -39,7 +41,8 @@ public class JsonAdaptedPersonTest {
     private static final String VALID_ROOMNUMBER =
                 BENSON.getRoomNumber() == null ? null : BENSON.getRoomNumber().toStringWDate();
     private static final String VALID_TELEGRAM = BENSON.getTelegram().toString();
-    private static final String VALID_BIRTHDAY = BENSON.getBirthday().toString();
```

Comment:
What's wrong with Benson's Bday lmao

Reply:
The string representation for the input of birthday is different as the string format of `toString()`, so in this case I just hard-coded an input for it.

But after your reminder, I realise we indeed have a way to let it look more standardised (do check out the latest code that I commit after this comment).

Reply:
Looks good!